### PR TITLE
Give credit where credit is due

### PR DESCRIPTION
**Pull request by brntbeer**
Noticed there was no license at all! Give credit where credit is due.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Brent Jackson
```
Comment (brntbeer) [inline LICENSE:3]: Didn't know if i should include Brent Jackson of Jackson Black!

**Comment by brntbeer** [comment]
The most gentle of bumps to you @jxnblk. Does this look good?

**Comment by jxnblk** [comment]
Thanks, but I don't like to needlessly add copyright info.

**Comment by brntbeer** [comment]
Not about adding copyright info, but if you ever want anyone to be able to legally use this, you'd need a license. 

To each their own though !

**Comment by jxnblk** [comment]
No, I get that, just this PR had weird formatting choices.

**Comment by brntbeer** [comment]
@jxnblk ah, yeah totally. auto-generated MIT license from the GitHub UI just keeps it at 80 characters.

¯_(ツ)_/¯

**Pull request closed without merging**